explainer.md

```diff
@@ -74,3 +93,27 @@ Creating an API that provides information about the real world has privacy impli
 *   The geometry of a known location may allow a site to identify the user's location
 
 In many ways these privacy implications are similar to video (camera) inputs and should be considered to have similar privacy implications. However, the implications are not necessarily equivalent. For example, users may consider their image to be more private than the layout of their room while a mesh of one's face may allow easier fingerprinting than RGB image processing.
+
+# API IDL
+```
+[SecureContext, Exposed=Window] interface XRHitResult {
+  readonly attribute Float32Array poseModelMatrix;
+};
+
+partial interface XRSession {
+  Promise<FrozenArray<XRHitResult>> requestHitTest(Float32Array origin, Float32Array direction, XRFrameOfReference frameOfReference);
+}
+```
+
+## API Details
+
+`hitTest` parameters
+*   origin - the origin of the ray as [x, y, z]
+*   direction - the direction of the ray as [x, y, z], normalized
```

Comment:
Normalization is a trivial thing to do, and relying on the user to input a normalized direction is going to be error prone. Best case scenario is they'll get a bad result and not understand why, worst case we'll have to check if the direction is normalized and error if not, in which case we might as well just normalize it ourselves. So I would advocate for speccing that we will normalize any non-zero-length direction vectors that are passed to the API. Here or anywhere else in WebXR for that matter.

Reply:
Done.